packages/testing/playwright/pages/parts/RunDataPanel.ts

```diff
@@ -0,0 +1,89 @@
+import type { Locator } from '@playwright/test';
+
+export class RunDataPanel {
```

Comment:
Same as the above for the comments, probably in this case a description of what the root element is in case it changes